lib/js-yaml/loader.js

```diff
@@ -1243,7 +1259,7 @@ function readAlias(state) {
   return true;
 }
 
-function composeNode(state, parentIndent, nodeContext, allowToSeek, allowCompact) {
+function composeNode(state, parentIndent, nodeContext, allowToSeek, allowCompact, allowPrefixTags) {
```

Comment:
If you are adding new function parameter, then **all** calls to that function must explicitly specify a **strictly-typed** argument for it. (i.e. only true or false, never undefined, null, 0, 1 etc)